NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -293,8 +307,8 @@
                           'keep HTXS_Higgs*',
                           'keep HTXS_njets30',
                           'keep Pileup*',
-                          'keep GenJet*',
-                          #'keep LHE*',
+                          'keep LHE*',
```

Comment:
This picks also these collections: 
```
LHEPdfWeight 
LHEReweightingWeight
LHEScaleWeight
```
as well as `LHEpart` and all single variables starting with `LHE_ `and `LHEWeight`
Is this intended? We should restrict the selection to only the variables that are really needed, especially in the AllEvents tree.
Also if I understand correctly your plan is to keep the LHEPart coll here only for development, right? If so it would be useful to keep it commented and uncomment it for your tests, so it will not be taken by people who are doing other studies right now.